style/js/shield_defs.js

```diff
@@ -1191,6 +1203,209 @@ export function loadShields(shieldImages) {
   );
   shields["NL:N"] = roundedRectShield("#eacb44", null, "black", 2, 0, null);
 
+  [
+    "Amsterdam",
+    "Den Haag",
+    "Rotterdam",
+    "Nijmegen",
+    "Parkstad",
+    "Zaanstad",
+  ].forEach(
+    (city) =>
+      (shields["NL:S:" + city] = {
+        backgroundImage: shieldImages.shield40_nl_city_2,
+        textColor: "black",
+        padding: {
+          left: 3,
+          right: 3,
+          top: 4,
+          bottom: 5,
+        },
+      })
+  );
+
+  // Germany
+  shields["BAB"] = {
+    backgroundImage: shieldImages.shield40_de_2,
+    textColor: "white",
+    padding: {
+      left: 3,
+      right: 3,
+      top: 4,
+      bottom: 5,
+    },
+  };
```

Comment:
German "BAB" shields contain "A 7" in `ref`, but should be displayed as "7" - the design implies that it's an Autobahn ("A").
So I'd propose to check if the `ref` starts with "A #" and if it is, then use "#", otherwise warn and use the full ref?

Reply:
Please see the discussion on this topic on #149

In short, I feel this would be a workaround for something that's essentially a mapping problem.  If the "A" is not included in the highway shield, then it should not be included in the `ref` tag on the road's highway relation.

Reply:
>  If the "A" is not included in the highway shield, then it should not be included in the ref tag on the road's highway relation

While it's not part of the sign, it's part of the name of the road, in addition to being an Autobahn.
In spoken communication (including radio) you say "Autobahn A 1". On some roadside safety markers it apparently says "A1" or it might just say "1" - there seem to be changes over time or regionally.

So, I have a hard time deciding this (and so does the German community apparently).

What makes this worse is that OSM seems to differentiate between references for [relations](https://wiki.openstreetmap.org/wiki/Key:ref#Examples_on_relations) and [ways](https://wiki.openstreetmap.org/wiki/Key:ref#Examples_on_ways):
- Relations: Machine readable and meant to be used together with `network`; "bare number"
- Ways: Human readable ("What's on the road")

Regardless of the above, with this being a `ref[erence]`, I think "A 7" is most clear & descriptive.

I also don't think bare numbers will work globally and as long as it's consistent within the network, it'll be fine.
If we define `network=DE:BAB` to have the "A " prefix in `ref`, that's probably not too bad?
The consumer of the data must be aware of the fact that "DE:BAB" means "A " prefix regardless (either to remove it, or to add it).
There's special code required to handle this shield in Americana anyway.

(Note, however, that I'd agree that `traffic_sign=DE:405:[number]` should not have the "A " prefix, as it's specifically meant for traffic-signs and shields)

Reply:
You're mixing up two separate taggings.  On ways, network and ref information are packed into the ref tag.  I.e. `ref=A1`.  The "A" is the network, and the "1" is the route number.  It's the same in the US, ways on US interstate highways have `ref=I-95` for Interstate 95.  Route relations are different because network and ref are separately tagged, so we have `network=US:I` and `ref=95` for the same road.  Note the difference in ref tagging.

IMO, what would make sense in Germany is to have `ref=A1` on the ways, and `network=DE:BAB` + `ref=1` on the relation.  That would be consistent with the preponderance of global usage.  We're waiting to see what the German mapping community comes up with, but at this point, prominent members of the German mapping community are even going so far as to argue against having route relations at all.  So in that light, it would be inappropriate for us to craft workarounds just to make the shields render in a certain way while the community is still deciding how to handle route tagging, and whether to even tag them at all.